src/api/app/assets/stylesheets/webui2/staging-workflow.scss

```diff
@@ -14,3 +14,15 @@
   }
 }
 
+ul .table-list-group-item {
+  padding: 0.5rem 0.75rem;
+  background-color: transparent;
+}
+
+.collapse-link[aria-expanded="false"] .less{
```

Comment:
I talked with @Ana06 and we synced on the collapse component code. @bgeuken and I came up with [this](https://github.com/openSUSE/open-build-service/pull/6265/files#diff-0b41ac7c12f6d6a9a017c59dd6725299). Depending on the order in which this PR and #6265 are merged, one PR will need to rebase and adapt its code a bit to align.